### PR TITLE
Update steps on how to recover pswd

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1802.surge.sh

Updated pages:
- [Recover admin user and password](https://neo4j-docs-operations-1802.surge.sh/operations-manual/5/authentication-authorization/password-and-user-recovery)

File: modules/ROOT/pages/authentication-authorization/password-and-user-recovery.adoc [hunk 1]
```diff
@@ -25,28 +25,20 @@ bin/neo4j stop
...
-This can be achieved by either:
 
-* Temporarily commenting out the `server.default_listen_address` parameter:
+. Block network connections during the recovery phase, so users can connect to Neo4j only via `localhost`.
+This can be achieved by temporarily commenting out the `server.default_listen_address` parameter and providing the specific localhost value:
 +
 [source, properties]
 ----
-#server.default_listen_address=<your_configuration>
+server.default_listen_address=<your_configuration>
```
Comment (OliviaYtterbrink) [inline modules/ROOT/pages/authentication-authorization/password-and-user-recovery.adoc:34]: `<your_configuration>` does not look like a `specific localhost value`

Comment (NataliaIvakina) [inline modules/ROOT/pages/authentication-authorization/password-and-user-recovery.adoc:34]:
```suggestion
server.default_listen_address=127.0.0.1
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1802.surge.sh

Updated pages:
- [Recover admin user and password](https://neo4j-docs-operations-1802.surge.sh/operations-manual/5/authentication-authorization/password-and-user-recovery)

**Review by OliviaYtterbrink — APPROVED** [review]

**Pull request merged**